src/cmdtab.c

```diff
@@ -1604,6 +1628,11 @@ static int RunCmdTab(handle instance, u16 *args)
 	DWM_WINDOW_CORNER_PREFERENCE corners = DWMWCP_ROUND;
 	DwmSetWindowAttribute(Switcher, DWMWA_WINDOW_CORNER_PREFERENCE, &corners, sizeof corners);
 
+	// Ensure the drawing is initialized, it might be instantly used in OnWindowPaint()
+	UpdateApps();
+	ResizeSwitcher(); // NOTE Must call ResizeSwitcher after UpdateApps, before RedrawSwitcher & before ShowSwitcher
+	RedrawSwitcher();
```

Comment:
#15 might be fixed by either this, or the GDI updates. I still can't repro the bug, so not sure ..